x-pack/packages/ml/date_picker/src/services/full_time_range_selector_service.ts

```diff
@@ -35,7 +35,8 @@ export async function setFullTimeRange(
   toasts: ToastsStart,
   http: HttpStart,
   query?: QueryDslQueryContainer,
-  excludeFrozenData?: boolean
+  excludeFrozenData?: boolean,
+  path: string = '/internal/file_upload/time_field_range'
```

Comment:
We could make this a bit safer by making the type a union of expected paths `'/internal/file_upload/time_field_range' | '/api/ml/fields_service/time_field_range'`

Reply:
Good idea, updated in 2156227.